client.go

```diff
@@ -52,7 +52,7 @@ import (
 	"github.com/containerd/containerd/namespaces"
 	"github.com/containerd/containerd/pkg/dialer"
 	"github.com/containerd/containerd/platforms"
-	"github.com/containerd/containerd/plugin"
+	"github.com/containerd/containerd/plugins"
```

Comment:
Maybe this should be a separate commit? For ease of review.

Reply:
I split the commits, its still not super easy by number of files, but the diffs are much easier to look at.